### PR TITLE
Display error alert box on supp app if form is invalid #164615215

### PR DESCRIPTION
**Pull request by chrisdolendo**

### CODE REVIEW

File: app/javascript/components/supplemental_application/SupplementalApplicationContainer.js
Comment (akegan) [inline app/javascript/components/supplemental_application/SupplementalApplicationContainer.js:134]: @chrisdolendo looks like coverage on this PR drops the coverage for the repo. Are you up for adding some more test coverage to the PR? maybe see if you can add a snapshot test for this behavior?

**Review by akegan — CHANGES_REQUESTED** [review]
@chrisdolendo QA looks good to me so far, but could you add some review instructions to the ticket? https://www.pivotaltracker.com/story/show/164615215.

Please also add a test for this change (see comment below)

File: app/javascript/components/supplemental_application/SupplementalApplicationContainer.js
Comment (chrisdolendo) [reply app/javascript/components/supplemental_application/SupplementalApplicationContainer.js:134]: Code climate is now happy :)

**Comment by akegan** [comment]
@chrisdolendo not sure if it's related to your changes at all, but it looks like I'm still able to click the status dropdown and make a change that way if the form state is invalid.

Edit: looks like that's the behavior on master too. let me know if you think it's a quick enough fix to make on this PR or if you'd like me to open up a new ticket

**Review by akegan — APPROVED** [review]

**Comment by akegan** [comment]
@chrisdolendo good to go to merge once you fix merge conflicts

Follow-up ticket: https://www.pivotaltracker.com/story/show/166453291

**Comment by akegan** [comment]
@chrisdolendo this should be good to merge once you fix merge conflicts!

**Comment by chrisdolendo** [comment]
Fixed, just waiting for tests to pass before merging!

**Pull request merged**
